includes/class-wc-stripe-webhook-handler.php

```
@@ -1578,4 +1583,141 @@ private function get_payment_method_type_from_charge( $charge ) {
+
+		WC_Stripe_Logger::info(
+			'Agentic checkout.session.completed received.',
+			[
+				'session_id'        => $checkout_session->id ?? 'unknown',
+				'payment_intent_id' => $payment_intent_id ?? 'unknown',
+			]
+		);
+
+		// Idempotency: look up existing order by payment intent ID.
+		if ( $payment_intent_id ) {
+			$existing_order = WC_Stripe_Helper::get_order_by_intent_id( $payment_intent_id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'process_checkout_session_completed'. (undefined)

(StaticAccess)